libdirections-onboard/src/main/java/com/mapbox/navigation/route/onboard/MapboxOnboardRouter.kt

```diff
@@ -57,26 +60,28 @@ class MapboxOnboardRouter(
     private val gson = Gson()
 
     init {
-        if (config.tilePath.isNotEmpty()) {
-            val tileDir = File(config.tilePath)
+        val directoryVersion = "Offline/${options.tilesUri.host}/${options.version}/tiles"
+        val tileFilePath = File(applicationContext.filesDir, directoryVersion).absolutePath
```

Comment:
This prevents users from providing a custom directory path for the tiles, for example in the external storage.

Reply:
thanks for this @LukasPaczos. asked in standup and we want to support custom file paths. 

added it back in.